### PR TITLE
Fixes/rebalences Regen coma to not always fake death

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl:
balance: The fake death from regenerative coma now only activates if the stealth is 2 or greater, previously all this changed was forcing a death-gasp, and the fake death always happened. 
/:cl:

You now need stealth 2 to trigger the fake death, else you go to sleep. Considering a slight rate nerf too ~20%

### CODE REVIEW

**Comment by Borisvanmemes** [comment]
iirc at high stealth it also makes you LOOK dead, along with the death grasp, but not too sure on that

**Comment by XDTM** [comment]
The fake death part is intended to happen as a part of the base package, but this works fine

**Comment by XDTM** [comment]
You need to add a form of unconsciousness for non-fake death coma or you remove the downside completely

**Comment by AnturK** [comment]
I'm confused about this change, doesn't this make this a free very strong heal ?

**Comment by Ratvar** [comment]
It's already a very strong heal. Thought, nothing stops anyone from spearing your head off before you regenerate to space your brain, like they should do anyway.

**Comment by ExcessiveUseOfCobblestone** [comment]
Can you explain what this does better in the CL?

Basically it makes it so you don't fall into a pretend death unless stealth is high, meaning it otherwise won't be as effective?

Basically a nerf unless you buff stealth at which point nothing has changed?

**Comment by XDTM** [comment]
I guess the intention is to cause normal unconsciousness without stealth and true fakedeath once the threshold is met.
The issue is that he didn't replace fakedeath with classic uncounsciousness so this would be a massive buff as long as you don't reach the stealth threshold.

**Comment by ExcessiveUseOfCobblestone** [comment]
okay now I understand lol

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 1]
```diff
@@ -251,6 +253,8 @@
 		M.emote("deathgasp")
 	if(A.properties["stealth"] >= 2)
 		M.fakedeath("regenerative_coma")
+	else if(intentionalsleep = FALSE)
```
Comment (XDTM) [inline code/datums/diseases/advance/symptoms/heal.dm]: ==

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 2]
```diff
@@ -260,6 +264,9 @@
 		return
 	active_coma = FALSE
 	M.cure_fakedeath("regenerative_coma")
+	if(intentionalsleep = FALSE)
```
Comment (XDTM) [inline code/datums/diseases/advance/symptoms/heal.dm]: ==

**Comment by XDTM** [comment]
I'm not sure you even need those intentional sleep checks. In any case, i'd suggest using Unconsious instead of Sleeping, because otherwise you're setting the healing to 25% of what it used to be due to the multiplier, while Unconscious has a 90% (which can be bumped to 100% potentially)

**Comment by ExcessiveUseOfCobblestone** [comment]
lol

**Pull request closed without merging**